packages/stream-model-handler/src/views-utils.ts

```diff
@@ -0,0 +1,22 @@
+import type { JSONSchema } from 'json-schema-typed/draft-2020-12'
+import { ModelViewsDefinition } from '@ceramicnetwork/stream-model'
+
+const SUPPORTED_VIEW_TYPES = [
+  'documentAccount'
+]
+
+export class ViewsValidation {
+  public validateViews(
+    views: ModelViewsDefinition,
+    schema: JSONSchema.Object
+  ) {
+    Object.entries(views).map(([key, value]) => {
```

Comment:
```suggestion
    Object.entries(views).forEach(([key, value]) => {
```

you do not really return anything here